packages/ddl-shim/src/ddl2.ts

```diff
@@ -288,9 +291,14 @@ export interface IView {
 //  DDL  ======================================================================
 export interface Schema {
     version: "0.0.22";
+    createdBy: {
+        name: string;
+        version: string;
+    };
     datasources: DatasourceType[];
     dataviews: IView[];
-    properties: IWidgetProperties;
+    properties?: IProperties;
+    hipieProperties?: IProperties;
```

Comment:
@drealeed here is a placeholder you can put "any" info in and the marshaller will persist (and ignore).